azerothcore/azerothcore-wotlk
# fix(Scripts/Spell): fix Spinning Up — Mimiron phase4

##### CHANGES PROPOSED:
-  possible fix for visual part of spell Spinning Up on phase 4 Mimiron encounter
###### ISSUES ADDRESSED:  Closes #1611
##### TESTS PERFORMED: build
##### HOW TO TEST THE CHANGES:
    Enter Ulduar 10 or 25 players
    Travel to Mimiron and engage him
    See the VX-001 P2 and  P4

##### Target branch(es): Master
